model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -215,7 +215,7 @@ message ExecutableStagePayload {
   string input = 2;
 
   // Side Input PCollection ids.
-  repeated string side_inputs = 3;
+  map<string, string> side_inputs = 3;
```

Comment:
If you're planning to add the PTransform id in this PR: Luke suggested just using a repeated field of a new (triple) embedded message type since proto maps do not allow message keys.